lib/routes/home/widgets/payments_list/dialog/payment_details_dialog_description.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:breez_sdk/bridge_generated.dart';
+import 'package:breez_translations/breez_translations_locales.dart';
+import 'package:c_breez/utils/extensions/payment_extensions.dart';
+import 'package:flutter/material.dart';
+
+class PaymentDetailsDialogDescription extends StatelessWidget {
+  final Payment paymentInfo;
+
+  const PaymentDetailsDialogDescription({
+    super.key,
+    required this.paymentInfo,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final texts = context.texts();
+    final themeData = Theme.of(context);
+
+    final description = paymentInfo.extractTitle(texts);
```

Comment:
I was suggesting to use description as it is here.

Reply:
Yep, I did it.